modules/internal/NetworkAtomics.chpl

```diff
@@ -41,7 +41,7 @@ module NetworkAtomics {
     }
 
     inline proc const read(order:memory_order = memory_order_seq_cst): bool {
-      pragma "insert line file info" extern externFunc("get", int(64))
+      pragma "insert line file info" extern externFunc("read", int(64))
         proc atomic_get(ref result:int(64), l:int(32), const obj:c_void_ptr): void;
```

Comment:
Lets update this name too. i.e. `atomic_get` => `atomic_read`. Same for `atomic_put` below

Reply:
Oh, sorry I missed that.  Will do.  What would you think of changing all those internal runtime interface functions to something like `commAtomicFn`?  None of them actually need to be named for the operation they perform, since they're internal to the individual op functions.

Reply:
I like keeping them named

Reply:
Done.